music/genres.rb

```diff
@@ -0,0 +1,16 @@
+require_relative '../classes/items'
+class Genres < Items
+  attr_reader :id, :items
+  attr_accessor :name
+
+  def initialize(name)
+    @id = rand(1..1000)
+    @name = name
+    @items = []
+  end
+
+  def add_item(item)
+    @items << item
+    item.add_genre = self
+  end
+end
```

Comment:
There's no option for creating a new Genre on the menu. When I list Genres I see album names. The Genre class is its own entity therefore it should have its own data that is separate from the album and at the same time associated with the album.

![image](https://user-images.githubusercontent.com/38088818/190639661-09d03036-53fa-4a26-986a-fc151366852e.png)

- You can add an option to create a new genre or choose from existing genres when creating a new album. Then list all genres from the Genre class.
- I should be able to list or create something like Hip Hop, Pop, Rock, RnB, Soul, Jazz, etc. from the Genres class.

![image](https://user-images.githubusercontent.com/38088818/190641106-acbdd1b2-e79e-4c25-95f8-66a6e6d6d17c.png)